server/.env.example

```diff
@@ -1,5 +1,6 @@
-NEGATIVE_QUERIES="china censorship side:theguardian.com, china taiwan site:theguardian.com, china genocide site:theguardian.com, china human rights violation site:theguardian.com"
-POSITIVE_QUERIES='site:chinadaily.com.cn when:7d'
+NEGATIVE_QUERIES="example country site:wikipedia.org, example topic site:wikipedia.org, example censorship site:wikipedia.org"
+POSITIVE_QUERIES='site:stackoverflow.com when:7d'
+CENSOR_REQUIREMENTS="example, world, today"
```

Comment:
how about we just use `CHANGEME` as the value for all variables? haven't tried it but i don't think these examples would be any better than `CHANGEME`